site/get-started.markdown

```diff
@@ -0,0 +1,137 @@
+  - [Join the community](#join-the-community)
+  - [Next steps](#next-steps)
+
+## Set up Haskell dev environment
+
+### GHCup: universal installer
+
+[GHCup](https://www.haskell.org/ghcup/#) is a universal installer for Haskell that will install for you everything you need to program in Haskell, and then will also help you manage those installations in the future (update, switch versions, ...).
+
+Follow instructions at [GHCup webpage](https://www.haskell.org/ghcup/#) to install GHCup. Then, use it to install the Haskell Toolchain, which consists of:
+
+1. **GHC** -> Haskell compiler. We will use it below to run our examples, but in practice, you will mostly be using a build tool like `cabal` or `Stack` to build your code, instead of `GHC` directly.
+2. **HLS** -> Haskell Language Server -> You won't use this directly, instead your code editor will use it in the background to provide you with a great experience while editing Haskell code.
+3. **cabal** -> Haskell build tool -> You will use this to structure your Haskell projects, build them, run them, define dependencies, ...
+4. **Stack** -> Haskell build tool -> alternative to `cabal`
+
+
+<div class="bs-callout bs-callout-info">
+  <p>
+    <h4>cabal and Stack -> which one should I install?</h4>
+    We recommend installing both. Most Haskell projects can be built using Cabal, but some might require Stack. Installing both guarantees that you can use either, and while following the tutorial/book you can use whatever they recommend.
+  </p>
+</div>
+
+### Editor
+**Visual Studio Code** is a popular choice with well supported editor integration. Install the [Haskell extension](https://marketplace.visualstudio.com/items?itemName=haskell.haskell) and you are all set. It should work out of the box and use your installation of HLS.
+
+Of other editors that have good Haskell extensions, the most popular ones are Vim and Emacs.
+
+## Running first lines of code
+
+We have everything set up, let's use it!
+
+`GHC` brings an interactive interpreter called `GHCi` together with it, which is great for playing with Haskell and trying things out, so let's give it a spin.
+
+Run `ghci`, which should start a new prompt for you.
+
+Let's do a simple calculation to check Haskell's computing capabilities:
+```
+> 6 + 3^2 * 4
+42
+```
+
+42 is a nice even number, but what about the first 10 even numbers after it?
+```
+> take 10 (filter even [43..])
+[44,46,48,50,52,54,56,58,60,62]
+```
+
+What is the sum of those?
+```
+> sum it
+530
+```
+**NOTE**: We used a special feature of GHCi here, which is a special `it` variable that remembers the result of the last expression.
+
+Great, you got the first taste of Haskell! Now let's get to running a real program.
+
+## Writing your first Haskell program
+
+In your editor, create a new file named `hello.hs`.
+
+Write the following in it:
+```hs
+main = do
+  putStrLn "Hello, everybody!"
+  putStrLn ("Please look at my favorite odd numbers: " ++ filter odd [10..20])
```

Comment:
Wait, this doesn't like like it will compile, `filter odd [10..20]` isn't of `String` type. perhaps add a `let` and `show`?

Reply:
Ouf, fixed. thanks!